onboarding_project/squad_pantry_app/admin.py

```diff
@@ -0,0 +1,33 @@
+from django import forms
+from django.forms import BaseInlineFormSet
+from django.contrib import admin
+from squad_pantry_app.models import Dish, Order, OrderDishRelation, SquadUser
+
+
+class BaseOrderDishFormset(BaseInlineFormSet):
+    def clean(self):
+        filled_forms = 0
+        for form in self.forms:
```

Comment:
Let's refactor this as:
```py
try:
    filled_forms = [form for form in self.forms if form.cleaned_data]
except AttributeError:
    pass
else:
     if len(filled_forms) < 1:
        raise forms.ValidationError('Enter atleast One Dish')
```

__Why?__:
1. We don't have to manually keep track of `count`
2. Less verbose code
3. List comprehensions are faster than normal for loops. See [here](https://stackoverflow.com/questions/30245397/why-is-list-comprehension-so-faster)
4. The `else` clause in a `try..catch` is something specific to python. It's a convention we follow. See [here](https://docs.python.org/3/tutorial/errors.html)